MovieDB/MovieDB/Scenes/Dashboard/Cells/Category Item/CategoryItemCollectionViewCell.swift

```diff
@@ -0,0 +1,45 @@
+import UIKit
+import Reusable
+
+fileprivate enum ConstantCategoryItemCollectionCell {
+    static let fontSizeCategoryTitle = CGFloat(12)
+}
+
+final class CategoryItemCollectionViewCell: UICollectionViewCell, NibReusable {
+    @IBOutlet private weak var categoryLabel: UILabel!
+    
+    var viewModel: CategoryItemCollectionViewCellViewModel! {
+        didSet {
+            categoryLabel.text = viewModel.title
+        }
+    }
+    
+    override var isSelected: Bool {
+        didSet {
+            categoryLabel.do {
+                if isSelected {
+                    $0.backgroundColor = AppColor.puertoRico
+                    $0.font = UIFont(name: AppFont.poppinsSemiBold,
+                                     size: ConstantCategoryItemCollectionCell.fontSizeCategoryTitle)
+                    $0.textColor = AppColor.charade
+                } else {
+                    configCell()
+                }
+            }
+        }
+    }
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        configCell()
```

Comment:
In the cell, we have 2 logic for 2 status of the cell, isSelected or not, right? 
So, the configCell() not clear meaning on this situation, and always note that `config` func should only call once at `init`. 
So, incase you should declare 2 func:
func updateUIWhenCellIsSelected() 
func updateUIWhenCellIsUnselected() 
Able to call them when init cell, isSelected was changed, ...
Read, understand and maintain easier